tests/function-controller/testsuite/teststep/secret.go

```diff
@@ -0,0 +1,43 @@
+package teststep
```

Comment:
What do you think about moving this file to `pkg`?

Reply:
There's already secret.go in pkg, but this file and that pkg/secret.go does similar things